charts/loki-distributed/templates/ingester/hpa.yaml

```diff
@@ -17,12 +18,24 @@ spec:
     - type: Resource
       resource:
         name: memory
+        {{- if (eq $apiVersion "autoscaling/v2") }}
+        target:
+          type: Utilization
+          averageValue: {{ . }}
```

Comment:
Is this a mistake? This field in other files is `averageUtilization`.

Reply:
Thanks, fixed.